analysers/experiments/cpu.py

```diff
@@ -181,6 +181,12 @@ def sortAndGet(field, asc):
 CassandraRDD = sc.cassandraTable(cassandraKeyspace, "trial_cpu_core")
 CassandraRDD.cache()
 
+nOfCores = CassandraRDD.select("cpu_mean") \
```

Comment:
@VincenzoFerme To have the correct indexes for the cores, and avoiding index out of range problems.